code/src/components/Summary.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Sunflower from "icons/sunflower.png";
+
+const Summary = ({ email, name, ageGroup, location }) => {
```

Comment:
I'm just got a tip in my code review,  that this way to destructing code can be used in the other components as well, instead of using the "const { propName, anotherPropName } = props", just to be consistent. So I thought I'd forward that too! :)